src/SFA.DAS.AANHub.Data/Repositories/NotificationsWriteRepository.cs

```diff
@@ -11,5 +11,9 @@ internal class NotificationsWriteRepository : INotificationsWriteRepository
 
     public NotificationsWriteRepository(AanDataContext aanDataContext) => _aanDataContext = aanDataContext;
 
-    public void Create(Notification notification) => _aanDataContext.Notifications.Add(notification);
+    public async Task<Guid> Create(Notification notification, CancellationToken cancellationToken)
```

Comment:
If we initialise the Id in handler or where ever we are initialising notification entity then we need not return the Id here.